docs/04-guides/on-demand/store-video-asset-on-ipfs/01-dashboard.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: dashboard
+title: Dashboard
+metaTitle: Store video asset on IPFS
```

Comment:
A couple frontmatter fixes:
- [ ]  We can remove the `metaTitle` frontmatter field. This isn't actually a field docusaurus recognizes.
- [ ] Let's add `pagination_next: guides/some-guide` so it links to an appropriate next guide at the end of the doc. Right now it links to the api tab.

Reply:
I'll add pagination_next for now but I think we'll have to adjust this later as the other guides for on demand get added